resolwe_docker_images/common/build.env

```diff
@@ -0,0 +1,3 @@
+BASE_IMAGE=public.ecr.aws/genialis/resolwebio/base:ubuntu-22.04-14112023
+REPOSITORY=resolwebio/common
+TAG=3.2.0
```

Comment:
The latest common image is on higher version already: `public.ecr.aws/genialis/resolwebio/common:4.1.1`

Reply:
Changed to `4.1.1`.